src/to-be-visible.js

```diff
@@ -0,0 +1,39 @@
+import { matcherHint } from 'jest-matcher-utils';
+import { mergeAll } from 'ramda';
+
+import { checkReactElement, printElement } from './utils';
+
+function isStyleVisible(element) {
+  const style = element.props.style || {};
+  const { display = 'flex', opacity = 1 } = Array.isArray(style) ? mergeAll(style) : style;
```

Comment:
Use `StyleSheet.flatten` to calculate flat `style` object.

Reply:
Good catch, I've applied this suggestion in [this commit](https://github.com/testing-library/jest-native/pull/73/commits/2335a11668f03b87d8cc1287096b6dcfbf378eb3).